roles/tackle/tasks/main.yml

```diff
@@ -885,6 +885,22 @@
     - (pathfinder_delete_db_volume|bool)
     - (pathfinder_pod.resources|length) == 0
 
+- name: "Remove RHSSO Subscription"
+  k8s:
+    state: absent
+    api_version: operators.coreos.com/v1alpha1
+    kind: Subscription
+    label_selectors:
+    - operators.coreos.com/rhsso-operator.openshift-mta =
+
```

Comment:
Is this ok here?
- indentation?
- no value for the equal?

Reply:
It looks funny, but it is correct. the label contains an empty string for the value. ansible/k8s loses it's mind if you try to give it an empty string with `""` or `''` because it escapes the quotes.
  labels:
    olm.managed: "true"
    operators.coreos.com/rhsso-operator.openshift-mta: ""

Reply:
indetation adjusted to appease the linter.